system/jlib/jsocket.cpp

```diff
@@ -3965,6 +4006,45 @@ class CSocketSelectThread: public CSocketBaseThread
         basesize = n;
     }
 
+    bool checkSocks()
+    {
+        bool ret = false;
+        ForEachItemIn(i,items) {
```

Comment:
Ok, my policy is that if I touch a method I correct the formatting at that point. That sometimes extends to if I touch a class.